stakepooldclient/stakepooldclient.go

```diff
@@ -731,3 +734,41 @@ func (s *stakepooldManager) CrossCheckColdWalletExtPubs(ctx context.Context, dcr
 	}
 	return nil
 }
+
+func (s *stakepooldManager) GetFeeAddress(ctx context.Context, txHash *chainhash.Hash, signature string) (*pb.GetFeeAddressResponse, error) {
+	var err error
+	responses := make([]*pb.GetFeeAddressResponse, len(s.grpcConnections))
+	for i, conn := range s.grpcConnections {
+		client := pb.NewStakepooldServiceClient(conn)
+		responses[i], err = client.GetFeeAddress(ctx, &pb.GetFeeAddressRequest{
+			Hash:      txHash.String(),
+			Signature: signature,
+		})
+		if err != nil {
+			return nil, fmt.Errorf("GetFeeAddress RPC failed on stakepoold instance %s: %v", conn.Target(), err)
+		}
+	}
+	return responses[0], nil
+}
+
+func (s *stakepooldManager) PayFee(ctx context.Context, txHash *chainhash.Hash, votingWIF *dcrutil.WIF, feeTx *dcrutil.Tx) (*pb.PayFeeResponse, error) {
+	feeTxBuffer := new(bytes.Buffer)
+	feeTxBuffer.Grow(feeTx.MsgTx().SerializeSize())
+	err := feeTx.MsgTx().Serialize(feeTxBuffer)
+	if err != nil {
+		return nil, err
+	}
+	responses := make([]*pb.PayFeeResponse, len(s.grpcConnections))
+	for i, conn := range s.grpcConnections {
+		client := pb.NewStakepooldServiceClient(conn)
+		responses[i], err = client.PayFee(ctx, &pb.PayFeeRequest{
+			TicketHash: txHash.String(),
+			VotingWIF:  votingWIF.String(),
+			FeeTx:      feeTxBuffer.Bytes(),
+		})
+		if err != nil {
+			return nil, fmt.Errorf("GetFeeAddress RPC failed on stakepoold instance %s: %v", conn.Target(), err)
+		}
+	}
+	return responses[0], nil
```

Comment:
2 concerns here
- Calling `PayFee` on multiple backends would cause multiple fee tx broadcast attempts. The broadcast could fail on any of those backends, no?
- If `PayFee` errors on one backend, the fee tx might have been broadcasted by another backend. Should probably ignore errors if the `PayFee` call to the first backend succeeds.